app/views/questions/new.html.haml

```diff
@@ -25,9 +25,6 @@
       %button.save_fields_button Save current options
       %br
       
-      =f.label :answer, 'Answer:'
```

Comment:
Don't we still need these in case there is an answer?

Reply:
We'd established earlier that since not every question would have an explicit answer, it didn't make sense to have it as an option on the form. 

Also, my understanding is that the Question model doesn't currently value the correctness of answers to begin with, so I'm not sure that we'd need it unless we decide to build something new that checks for right/wrong student answers

Reply:
Got it! Sorry about that :)